src/backend/marsha/core/tests/test_utils_medialive_utils.py

```diff
@@ -320,7 +325,301 @@ def test_create_medialive_channel(self):
+                                        "Syntax": "DEFAULT",
+                                        "TemporalAq": "ENABLED",
+                                        "TimecodeInsertion": "DISABLED",
+                                        "NumRefFrames": 1,
+                                        "AfdSignaling": "NONE",
+                                    }
+                                },
+                                "Height": 480,
+                                "Name": "video_854_480",
+                                "ScalingBehavior": "DEFAULT",
+                                "Width": 854,
+                                "RespondToAfd": "NONE",
+                                "Sharpness": 50,
+                            },
+                            {
+                                "CodecSettings": {
+                                    "H264Settings": {
+                                        "AdaptiveQuantization": "HIGH",
+                                        "Bitrate": 3300000,
+                                        "ColorMetadata": "INSERT",
+                                        "EntropyEncoding": "CABAC",
+                                        "FlickerAq": "ENABLED",
+                                        "FramerateControl": "SPECIFIED",
+                                        "FramerateDenominator": 1100,
+                                        "FramerateNumerator": 30000,
+                                        "GopBReference": "ENABLED",
+                                        "GopNumBFrames": 2,
+                                        "GopSize": 1,
+                                        "GopSizeUnits": "SECONDS",
+                                        "Level": "H264_LEVEL_4_1",
+                                        "LookAheadRateControl": "HIGH",
+                                        "ParControl": "INITIALIZE_FROM_SOURCE",
+                                        "Profile": "HIGH",
+                                        "RateControlMode": "CBR",
+                                        "SceneChangeDetect": "ENABLED",
+                                        "SpatialAq": "ENABLED",
+                                        "Syntax": "DEFAULT",
+                                        "TemporalAq": "ENABLED",
+                                        "TimecodeInsertion": "DISABLED",
+                                        "NumRefFrames": 1,
+                                        "AfdSignaling": "NONE",
+                                    }
+                                },
+                                "Height": 720,
+                                "Name": "video_1280_720",
+                                "ScalingBehavior": "DEFAULT",
+                                "Width": 1280,
+                                "RespondToAfd": "NONE",
+                                "Sharpness": 50,
+                            },
+                        ],
+                    },
```

Comment:
I know this isn't related to your PR, but I don't get this test.
It seems like we're mocking something, and we're only testing that the mocks works, but I surely miss something.

Reply:
We are mocking the call to AWS. The mock library check if the request contains valid parameter and we are checking if the request contains what we expect.

Reply:
https://botocore.amazonaws.com/v1/documentation/api/latest/reference/stubber.html

Reply:
OK, thanks :pray: